burn-book/src/import/onnx-model.md

```diff
@@ -1,137 +1,210 @@
-exported model weights before. PyTorch, for instance, lets you save model weights using its
-`torch.save()` function. Yet, to port this model to another framework, you face the arduous task of
-manually recreating the architecture in the destination framework before loading in the weights. Not
-only is this method tedious, but it's also error-prone and hinders smooth interoperability between
-frameworks.
+As the field of deep learning continues to evolve, the need for interoperability between different
+frameworks becomes increasingly important. Burn, a modern deep learning framework in Rust,
+recognizes this need and provides robust support for importing models from other popular frameworks.
+This section focuses on importing
+[ONNX (Open Neural Network Exchange)](https://onnx.ai/onnx/intro/index.html) models into Burn,
+enabling you to leverage pre-trained models and seamlessly integrate them into your Rust-based deep
+learning projects.
 
-It's worth noting that for models using cutting-edge, framework-specific features, manual porting
-might be the only option, as standards like ONNX might not yet support these new innovations.
+## Why Import Models?
 
-## Enter ONNX
+Importing pre-trained models offers several advantages:
 
-[ONNX (Open Neural Network Exchange)](https://onnx.ai/onnx/intro/index.html) is designed to solve
-such complications. It's an open-standard format that exports both the architecture and the weights
-of a deep learning model. This feature makes it exponentially easier to move models between
-different frameworks, thereby significantly aiding interoperability. ONNX is supported by a number
-of frameworks including but not limited to TensorFlow, PyTorch, Caffe2, and Microsoft Cognitive
-Toolkit.
+1. **Time-saving**: Avoid the need to train models from scratch, which can be time-consuming and
+   resource-intensive.
+2. **Access to state-of-the-art architectures**: Utilize cutting-edge models developed by
+   researchers and industry leaders.
+3. **Transfer learning**: Fine-tune imported models for your specific tasks, benefiting from
+   knowledge transfer.
+4. **Consistency across frameworks**: Ensure consistent performance when moving from one framework
+   to another.
 
-### Advantages of ONNX
+## Understanding ONNX
 
-ONNX stands out for encapsulating two key elements:
+ONNX (Open Neural Network Exchange) is an open format designed to represent machine learning models.
+Key features include:
 
-1. **Model Information**: It captures the architecture, detailing the layers, their connections, and
-   configurations.
-2. **Weights**: ONNX also contains the trained model's weights.
+- **Framework agnostic**: ONNX provides a common format that works across various deep learning
+  frameworks.
+- **Comprehensive representation**: It captures both the model architecture and trained weights.
+- **Wide support**: Many popular frameworks like PyTorch, TensorFlow, and scikit-learn support ONNX
+  export.
```

Comment:
Not sure if I would highlight this if we don't support ONNX export ourselves 😅

Reply:
"Popular frameworks" =D

I am sure if we are as big as PyTorch, someone will come along and support it. My philosophy is that other frameworks support that because they are primarily training/experimentation and Burn will do well in training and inference as well. So there will be less need for export. 

Conversely, the popular frameworks do not support importing as Burn =D